index.d.ts

```diff
@@ -445,15 +450,6 @@ declare module 'react-native-health' {
     end: string
   }
 
-  export interface ElectrocardiogramSampleValue extends BaseValue {
```

Comment:
Hey, I think this should not be deleted

Reply:
@macelai very weird. I've merged master to my fork. It's showing "up to date" and no changes, but these lines are missing from my fork and not merging from master. I can add them back manually.

Reply:
@macelai Added it back.